lib/galaxy/tools/sort_collection_list.xml

```diff
@@ -1,6 +1,6 @@
 <tool id="__SORTLIST__"
       name="Sort Collection"
-      version="1.0.0"
+      version="1.0.1"
```

Comment:
Not necessary IMO, just breaks workflows using this tool.

Reply:
OK. Removed my last commit again.